app/templates/app/questions.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'app/_base.html' %}
+{% load static %}
+
+
+    {% block title %}
+        Questions
+    {% endblock %}
+
+    {% block content %}
+<!--could put if user.is_authenticated here possibly -->
+    <div id="questionPage" class="container">
+        <h1 class="text-muted">Questions</h1>
+        <br>
+            <h2 name="question" style="color: #003366" id="question">[[ questionPackage.question ]]</h2>
+<!--        <h3>[[ questionPackage.key ]]</h3>-->
+
+        <br>
+                <fieldset v-for="answer in questionPackage.answer_options" class="custom-control custom-radio">
+                    <input class="list-group-item" v-model="answerVal" :value="answer" type="radio" id="answer" name="response" @click="evaluateAnswer(answer)"><label class="label">[[ answer ]]</label><br>
+                </fieldset>
+            <br>
+            <hr class="rounded">
+                <h5 style="color: darkred">[[ message ]]</h5>
+            <br>
+            <button v-if="message" style="color: darkred" class="btn btn-outline-secondary" onClick="window.location.reload();">Next Question</button>
```

Comment:
Instead of reloading the window, you should add an event listener, create a method to get a new question, then clear your question data and reload it from the API response.  We only want to rely on the mounted lifecycle method on the very first load, and then have Vue handle the reactivity from there on out.